index.html

```diff
@@ -252,6 +252,8 @@ <h5>CONTACT US</h5>
               name="user_email"
               placeholder="Email Address"
             />
+            <span class="error"></span>
+            
```

Comment:
Kindly move your error message above the submit button. Also, your error message is supposed to come up after validation, not always as seen in the below image

![image](https://user-images.githubusercontent.com/12239821/175531793-dc7db94a-2447-4a13-bab0-0e719cd2387b.png)